src/main/scala/cats/effect/bio/BIO.scala

```diff
@@ -436,7 +440,7 @@ private[effect] abstract class IOLowPriorityInstances extends IOParallelNewtype
 
   implicit def ioSemigroup[E, A: Semigroup]: Semigroup[BIO[E, A]] = new IOSemigroup[E, A]
 
-  implicit def ioMonadError[E]: Bracket[BIO[E, ?], E] = new Bracket[BIO[E, ?], E] {
+  implicit def ioMonadError[E]: MonadError[BIO[E, ?], E] = new MonadError[BIO[E, ?], E] {
```

Comment:
What happened to `Bracket`?

Reply:
we've downgraded the version of cats-effect (for the purposes of compatibility within our stack), but I can bump it back up 👍